mesonbuild/dependencies/base.py

```diff
@@ -1477,10 +1480,23 @@ def _detect_dep(self, name: str, modules: T.List[T.Tuple[str, bool]], components
                     otherDeps += [x for x in tgt.properties['IMPORTED_LINK_DEPENDENT_LIBRARIES'] if x]
 
                 for j in otherDeps:
+                    is_abs_path = os.path.split(j)[0]
```

Comment:
Why is this variable named `is_abs_path` when `os.path.split` is used? Are you sure you didn't want to use `os.path.isabs()`?

Reply:
Thanks, I missed `isabs` in the `os.path` documentation somehow. But the variable is more or less what it claims to be: `os.path.split()` splits into a two-element `(dirname,basename)` tuple, so for absolute and relative paths the `[0]` element will evaluate to a truthy value, and not for bare arguments (e.g. `version` and `uuid`).

Reply:
No problem, that's what code review is for :)